packages/entity-browser/src/components/DetailView/DetailView.js

```diff
@@ -4,7 +4,9 @@ import DetailForm from './DetailForm'
 import syncValidation from '../../util/syncValidation'
 import {asyncValidate, AsyncValidationException} from '../../util/asyncValidation'
 
-import './styles.scss'
+if (__DEV__) {
```

Comment:
Your right, 'tocco-theme/src/ToccoTheme/theme.scss' has always to be required in main.js.